pkg/report/linux.go

```diff
@@ -1399,6 +1399,19 @@ var linuxOopses = append([]*oops{
 				report: compile("BUG: .*still has locks held!(?:.*\\n)+?.*{{PC}} +{{FUNC}}"),
 				fmt:    "BUG: still has locks held in %[1]v",
 			},
+			{
+				title:  compile("BUG: scheduling while atomic"),
+				report: compile("BUG: scheduling while atomic(.*)"),
+				fmt:    "BUG: scheduling while atomic in %[2]v",
+				alt:    []string{"BUG: scheduling while atomic%[1]v"},
```

Comment:
If it will result in `BUG: scheduling while atomic: syz-executor/ADDR` all the time, won't all such reports be put in only one bucket on syzbot? Or that'd be exactly what's needed?

Reply:
You are right. I was thinking about potential duplicates. But since we have only 2 open bugs of this type, I guess it's better to dup them manually to the new bugs.
Done.
PTAL